tests/src/backend/test_starknet.py

```diff
@@ -1,6 +1,32 @@
+from starkware.starknet.public.abi import get_selector_from_name
+
+from tests.utils.syscall_handler import SyscallHandler
+
+
 class TestStarknet:
     class TestGetEnv:
         def test_should_return_env_with_given_origin_and_gas_price(self, cairo_run):
             env = cairo_run("test__get_env", origin=1, gas_price=2)
             assert env["origin"] == 1
             assert env["gas_price"] == 2
+
+    class TestSaveTestStarknetJumpdests:
+        @SyscallHandler.patch("IAccount.write_jumpdests", lambda addr, data: [])
+        def test_should_save_jumpdests_to_storage(self, cairo_run):
+            valid_jumpdests = [(0x01, 0, 1), (0x10, 0, 1), (0x101, 0, 1)]
```

Comment:
no false case here
I think that the test would be more readable with an input defined like

```
valid_jumpdests = {
    0x1: True
    0x10: False
    0x101: True
}
```

and use something like the `dict_new` function for the core lib

```
func dict_new() -> (res: DictAccess*) {
    %{
        if '__dict_manager' not in globals():
            from starkware.cairo.common.dict import DictManager
            __dict_manager = DictManager()

        memory[ap] = __dict_manager.new_dict(segments, initial_dict)
        del initial_dict
    %}
    ap += 1;
    return (res=cast([ap - 1], DictAccess*));
}
```